libs/mrwebrtc-unityplugin/include/api.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "mrs_errors.h"
+#include "interop_api.h"
+#include "../src/log_helpers.h"
+
+extern "C" {
+
+using mrsResult = Microsoft::MixedReality::WebRTC::Result;
+
+/// Opaque handle to a native PeerConnection C++ object.
+using PeerConnectionHandle = void*;
+
+//
+// Native rendering
+//
+
+enum class VideoKind : int32_t {
+  kNone = 0,
+  kI420 = 1,
+  kARGB = 2,
+};
+
+struct TextureDesc {
+  void* texture{nullptr};
+  int width{0};
+  int height{0};
+};
+
+/// Signature of rendering method called by Unity.
+typedef void(MRS_CALL* VideoRenderMethod)();
+
+/// Create a native renderer and return a handle to it.
+MRS_API mrsResult MRS_CALL
+mrsNativeRenderer_Create(mrsRemoteVideoTrackHandle videoHandle) noexcept;
+
+/// Destroy a native renderer.
+MRS_API mrsResult MRS_CALL
+mrsNativeRenderer_Destroy(mrsRemoteVideoTrackHandle videoHandle) noexcept;
+
+//// Register textures for remote video and start rendering it.
+MRS_API mrsResult MRS_CALL
+mrsNativeRenderer_EnableRemoteVideo(mrsRemoteVideoTrackHandle videoHandle,
+                                    VideoKind format,
+                                    TextureDesc textures[],
+                                    int textureCount) noexcept;
+
+/// Clear remote textures and stop rendering remote video.
+MRS_API mrsResult MRS_CALL mrsNativeRenderer_DisableRemoteVideo(mrsRemoteVideoTrackHandle videoHandle) noexcept;
```

Comment:
It would be more consistent with the rest of the API if `mrsNativeRenderer_Create` returned a `mrsNativeRendererHandle` and the other methods took a `mrsNativeRendererHandle` rather than a `mrsRemoteVideoTrackHandle`. `mrsNativeRendererHandle` could just be a typedef of `NativeRenderer*` and you wouldn't need a global map of remote track to renderer. See the other mrs*Handle types, this is the same pattern used for them.